### ISSUE TITLE
New Psy Shell v0.10.8 throws error when calling help

### ISSUE THREAD

**Issue by me-kell** [bobthecow/psysh#675]
New Psy Shell v0.10.8 throws error when calling help

```
wget "https://github.com/bobthecow/psysh/releases/download/v0.10.8/psysh-v0.10.8.tar.gz"
tar -xzf psysh-v0.10.8.tar.gz
chmod +x psysh
./psysh

Psy Shell v0.10.8 (PHP 7.3.27-1~deb10u1 — cli) by Justin Hileman
>>> help
PHP Fatal error:  Cannot use '_HumbugBoxf1db15892a9e\parent' as class name as it is reserved in phar:///path/to/psysh/vendor/symfony/string/UnicodeString.php on line 141
```

**Comment by bobthecow** [comment]
This is an upstream issue with `php-scoper`, which is scoping the `parent` return type:

 - [humbug/php-scoper#465](https://github.com/humbug/php-scoper/issues/465)
 - [humbug/php-scoper#466](https://github.com/humbug/php-scoper/pull/466)

**Comment by shishtpal** [comment]
I too am facing this issue 😢

**Comment by shishtpal** [comment]
I too am facing this issue 😢

**Comment by bobthecow** [comment]
This is resolved in the next release. In the meantime, you can install via composer (or build your own phar) to get the fix!

**Comment by bobthecow** [comment]
This has been resolved in v0.10.11.
